packages/cli/core/src/commands/auth/login.ts

```diff
@@ -46,8 +47,18 @@ export default class Login extends CLICommand {
     Organization: ${cfg.organization}
     Region: ${cfg.region}
     Environment: ${cfg.environment}
-    Run auth:login --help to see the available options to log in to a different organization, region, or environment.
     `);
+    const hasOrgFlag = Boolean((await this.parse(Login)).flags.organization);
+    this.log(
+      dedent`
+    To changes organizations either:
+      * Run coveo auth:login --help to see the available options to log in to a different organization, region, or environment.
+      ${
+        hasOrgFlag
+          ? '* Run coveo config:set -o=theOtherOrg to run the next commands against theOtherOrg'
+          : ''
+      }`.trimEnd()
```

Comment:
Nitpick, But can we have the bullet list only if `hasOrgFlag` is set to true.
Otherwise it's a little bit weird to have this message:
```
To changes organizations either:
  * Run coveo auth:login --help to see the available options to log in to a different organization, region, or environment.
```

Reply:
And also, to keep consistent with other lists in the CLI, let's use the same symbol. I think we used bull character.